.coveragerc

```diff
@@ -5,6 +5,8 @@ exclude_lines =
     raise NotImplemented.
     return NotImplemented
     def __repr__
+    if TYPE_CHECKING:
+    ^\s*\.\.\.$
```

Comment:
I hope this is an okay way to handle the MR coverage percent check. This excludes the TYPE_CHECKING imports as well as any line that is only an ellipsis